### PR TITLE
test: observe a warning from a real TouchDesigner, not a fixture

### PR DESCRIPTION
**Pull request by 8beeeaaat**
Closes the one gap the release-test audit found before cutting v2.1.0.

## The gap

Warnings are the feature this release exists for, and they were **never observed against a live TouchDesigner**.

```
$ grep -n "warning" tests/integration/touchDesignerClientAndWebServer.test.ts
(no output)
```

All 96 warning assertions in the repo are one of two kinds:

- `tests/integration/nodeErrorsTool.test.ts` (66) — feeds the formatter a payload the test wrote itself
- `tests/python/test_node_errors_report.py` (30) — drives the parser against a stubbed `td`

Neither can tell you that TouchDesigner actually puts a missing file or a dangling reference on `warnings()` rather than `errors()`. That premise is the whole change, and nothing checked it.

The audit also showed the live suite gained **no new test** in this release range — 63 lines changed, zero `+it(` — it was only updated, for the sandbox collision (#216) and the message-format change.

## What this adds

Two different warning sources, so no single operator quirk can carry the test: a `selectTOP` pointing at a path that is not there, and a `moviefileinTOP` whose file is missing. Measured on 099.2025.33230 before writing the assertions:

```
errors:   (empty)
warnings: /…/bad_select:Warning:   Invalid path for node "/project1/does_not_exist" …
          /…/missing_movie:Warning: Failed to open file. …
```

The test reconciles rather than asserts a shape — the two operators it breaks are the two it expects back:

```ts
expect(warnings.map((w) => w.nodePath).sort()).toEqual([
    `${warnHost}/bad_select`,
    `${warnHost}/missing_movie`,
]);
expect(response.data?.warningCount).toBe(2);
```

and then the discriminating half:

```ts
expect(errors).toEqual([]);
expect(response.data?.errorCount).toBe(0);
expect(response.data?.hasErrors).toBe(false);
```

The implementation this release replaces read only `errors()`, so this exact scene came back **completely clean**. A build that skips `warnings()` returns an empty list and a zero count; a build that files these under errors fails the second block. Either way it fails here instead of reporting a healthy node.

It also pins the message-format change live (no repeated path prefix, no trailing `(<path>)`), `level: "warning"`, and that `incomplete` is false with `skippedStreams` empty — so the counts are a total rather than a floor.

## On the mutation check

Not done the usual way. Proving it by breaking the collector means reloading Python inside the running TouchDesigner, which has already cost this branch one long detour chasing stale module state. The discrimination rests on the measured raw streams above instead, which show the two messages arriving on `warnings()` with `errors()` empty.

## Verification

- [x] `tests/integration/touchDesignerClientAndWebServer.test.ts` — **25 passed** (was 24)
- [x] `tsc --noEmit`, `biome check` — clean (the one warning is pre-existing)

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: tests/integration/touchDesignerClientAndWebServer.test.ts [hunk 1]
```diff
@@ -542,6 +542,79 @@ describe("TouchDesigner Client E2E Tests", () => {
...
+		// cannot carry the test: a dangling operator reference and a file that
+		// is not there. Measured on 099.2025.33230 — both land on warnings()
+		// with errors() empty.
+		const setup = await tdClient.execPythonScript<{ result: unknown }>({
+			script:
+				`host = op('${warnHost}')\n` +
+				`m = host.create('moviefileinTOP', 'missing_movie')\n` +
+				`m.par.file = '/nonexistent/definitely_not_here.mov'\n` +
+				`s = host.create('selectTOP', 'bad_select')\n` +
+				`s.par.top = '/project1/does_not_exist'\n` +
```
[the remark this answers is not in the document]
Comment (8beeeaaat) [reply tests/integration/touchDesignerClientAndWebServer.test.ts]: Right, and fixed in `c643d56`. I measured it rather than take it on trust — with a `constantTOP` of that name present in the project:

```
warnings when /project1/does_not_exist EXISTS: (empty)
```

The warning does not merely change, it **disappears**: the Select TOP resolves, and the test receives one warning instead of two and fails its counts for a reason that has nothing to do with the collector under test.

The reference now points at a child path under the per-process sandbox, which the test owns and never creates. Confirmed live that the warning text is identical:

```
/…/warn_probe2/bad_select:Warning: Invalid path for node
  "/project1/warn_probe2/no_such_child" referenced by parameter "TOP"
```

Worth naming the pattern, because you have found it twice on this branch now: **this suite already had exactly this defect once** — the fixed `test_base_comp` sandbox name, where a run depended on state outside its own isolation and inherited the previous run's children. I fixed that in #216 and then reached outside the sandbox again in the very test written to close an audit gap. The comment now says so at the line that would tempt the next person to do it a third time.

**Comment by 8beeeaaat** [comment]
@codex review

Re-review against `c643d56`. The dangling reference now points inside the run-unique sandbox instead of at an invented `/project1` path. Verified live that a project containing that name makes the warning vanish entirely (not just differ), which is exactly the failure you described, and that the sandbox-internal reference produces an identical warning.
